mods/cnc/maps/nod04a/nod04a.lua

```diff
@@ -0,0 +1,320 @@
+XxxxTriggerFunctionTime = DateTime.Seconds(50)
+YyyyTriggerFunctionTime = DateTime.Minutes(1) + DateTime.Seconds(40)
+ZzzzTriggerFunctionTime = DateTime.Minutes(2) + DateTime.Seconds(30)
+
+NodCivilians = { { { 'c9' }, { spawnciv0.Location } }, { { 'c8' }, { spawnciv1.Location } }, { { 'c7' }, { spawnciv2.Location } }, { { 'c4' }, { spawnciv3.Location } }, { { 'c3' }, { spawnciv4.Location } }, { { 'c1' }, { spawnciv5.Location } }, { { 'c4' }, { spawnciv6.Location } }, { { 'c5' }, { spawnciv7.Location } }, { { 'c2' }, { spawnciv8.Location } } }
+NodCiviliansActors = {}
+
+Atk6TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'apc', Atk6Units, Atk6WaypointsPart1, Atk6WaypointsPart2, 
+	function(transport, cargo)
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end, 
+	function(unit)
+		IdleHunt(unit)
+	end)
+end
+
+Atk5TriggerFunction = function ()
+	if not Atk5TriggerSwitch then
+		Atk5TriggerSwitch = true
+		Reinforcements.ReinforceWithTransport(GDI, 'apc', Atk5Units, Atk5Waypoints, nil, 
+		function(transport, cargo)
+			transport.UnloadPassengers()
+			Utils.Do(cargo, function(actor)
+				IdleHunt(actor)
+			end)
+		end,
+		function(unit)
+			IdleHunt(unit)
+		end)
+	end
+end
+
+Atk1TriggerFunction = function()
+	Reinforcements.Reinforce(GDI, Atk1Units, Spawnpoint, 15, 
+	function(actor) 
+		Atk1Movement(actor)
+	end)
+end
+
+XxxxTriggerFunction = function()
+	if not XxxxTriggerSwitch then
+		Reinforcements.Reinforce(GDI, XxxxUnits, Spawnpoint, 15, 
+		function(actor) 
+			Atk2Movement(actor)
+		end)
+	end
+end
+
+YyyyTriggerFunction = function()
+	if not YyyyTriggerSwitch then
+		Reinforcements.Reinforce(GDI, YyyyUnits, Spawnpoint, 15, 
+		function(actor) 
+			Atk4Movement(actor)
+		end)
+	end
+end
+
+ZzzzTriggerFunction = function()
+	if not ZzzzTriggerSwitch then
+		Reinforcements.ReinforceWithTransport(GDI, 'apc', ZzzzUnits, Atk5Waypoints, nil, 
+		function(transport, cargo) 
+			transport.UnloadPassengers()
+			Utils.Do(cargo, function(actor)
+				IdleHunt(actor)
+			end)
+		end,
+		function(unit)
+			IdleHunt(unit)
+		end)
+	end
+end
+
+Atk3Movement = function(unit)
+	Utils.Do(Atk3Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Atk2Movement = function(unit)
+	Utils.Do(Atk2Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Atk1Movement = function(unit)
+	Utils.Do(Atk1Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+GcivMovement = function(unit)
+	Utils.Do(GcivWaypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+Atk4Movement = function(unit)
+	Utils.Do(Atk4Waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+InsertNodUnits = function()
+	Reinforcements.Reinforce(Nod, NodUnits, { UnitsEntry.Location, UnitsRally.Location }, 15)
```

Comment:
Could you split the entry and rallypoint locations for those a bit? It takes ages for all those units to enter, and they all huddle up around the same place, which doesn't look too good.